Responsive design for 280px to 376px width mobile devices

JiHaripriya opened the pull request.

JiHaripriya committed:
Responsive design for 280px to 376px mobile devices

JiHaripriya committed:
Delete unwanted files

JiHaripriya committed:
Add support to several mobile devices

JiHaripriya committed:
Added ipad pro dimensions to responsive css

JiHaripriya committed:
Add screenshots of different mobile views

Priyaqb commented:
- Apply mobile design below 767
- Give line heights for texts wherever needed.
- Give spacing after image in mobile view.
- flex property should be given to the items.
- Read more can be added as a link. input will come only inside form tag.
- Use proper all text.

JiHaripriya committed:
Footer made responsive to device width

JiHaripriya committed:
Removed unwanted files, added comments, and cleaned code

JiHaripriya commented:
1. Applied mobile design for the following breakpoints:
 - min-width 540px
 - 540px to 739px
 - 739px to 990px
2. Added line heights for texts in responsive CSS section.
3. Added spacing for image in mobile view.
4. Assigned flex property to items
5. Read more and contact us were made to hyperlinks.
6. Modified and added appropriate alternate text for images.
7. Added HTML entity for copyright.
8. Modified hover property appearance in mobile view.

JiHaripriya committed:
Fixed font issue

Priyaqb commented:
- Check the code `<h2 >Our Recent Projects</h3>`
- Remove flex from 

```
.column-align {
    display: flex;
    flex-direction: column;
    flex: 0 0 33%;
}
```

- Instead of giving common styles like `'.what-we-offer__content > div{`' please use class names.

JiHaripriya committed:
Made changes according to review on tag correction and class addition

JiHaripriya commented:
> * Check the code `<h2 >Our Recent Projects</h3>`
> * Remove flex from
> 
> ```
> .column-align {
>     display: flex;
>     flex-direction: column;
>     flex: 0 0 33%;
> }
> ```
> 
> * Instead of giving common styles like `'.what-we-offer__content > div{`' please use class names.

1. Code issues have been fixed and validated.
2. Removing flex from style.css will disorganize the intended element.
3. Replaced maximum number of `div` tags and inner selectors of `div` with class names.

JiHaripriya committed:
Removed unwanted spacing, stray div tag and intended CSS

JiHaripriya committed:
Changed div to class names
